pwa-devdocs/src/tutorials/pwa-studio-fundamentals/index.md

```diff
@@ -17,6 +17,10 @@ During the course of the tutorials, you will create a new storefront application
 -   Node >= 10.14.1
 -   Yarn (recommended) or NPM
 
+{: .bs-callout .bs-callout-info}
+If you are using Node 12, you may see a deprecation warning in your logs.
```

Comment:
I would be specific about this log, rather than just say "there might be a deprecation warning". That way other deprecation notices, should they arise, are noticed and not ignored.

I should have documented this originally but this is the "expected" warning:

[![Image from Gyazo](https://i.gyazo.com/e42662101547183f0737bef7e7cdbc01.png)](https://gyazo.com/e42662101547183f0737bef7e7cdbc01)